keyboards/ergodash/rev1/keymaps/totuus1157/keymap.c

```diff
@@ -0,0 +1,177 @@
+#include QMK_KEYBOARD_H
+
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
```

Comment:
We recommend using an enum for layer names, as they auto increment.  Especially in this case, as you're adding 13 unused layers, by setting adjust to layer 16.
```suggestion
enum custom_layers {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```